<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatLines</class>
 <widget class="QMainWindow" name="PatLines">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PatLines</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1181</width>
      <height>701</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>720</y>
      <width>231</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Solid Line            ________</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dotted Line        . . . . . . . . .</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dashed Line       - - - - - - - -</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DashDot Line     _ . _ . _ . _</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thick Line            =======</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelPat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pattern:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelX1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>720</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>x1:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>720</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditX1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>720</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelY1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>720</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>y1:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditY1">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>720</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelX2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>720</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>x2:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditX2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>720</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelY2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>720</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>y2:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditY2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>720</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="drawPB">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>720</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clearPB">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>720</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>720</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Or, use LMB to draw</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>720</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>720</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
